aea/mail/base.py

```diff
@@ -267,7 +268,7 @@ def __init__(
         to: Address,
         sender: Address,
         protocol_id: ProtocolId,
-        message: bytes,
+        message: Union[Message, bytes],
```

Comment:
We accept both; because envelopes on the connection side do not have access to protocol serializer objects

Reply:
why not just `Message`? After the change suggested above.

Reply:
Because the change above is not straightforward and has its own downsides.